README.rst

```diff
@@ -206,9 +152,14 @@ the workflow and the expected outputs:
         steps:
           - environment: 'reanahub/reana-env-root6'
             commands:
-            - root -b -q 'code/gendata.C(20000,"data.root")'
-            - root -b -q 'code/fitdata.C("data.root","plot.png")'
-
+            - mkdir -p results
+            - root -b -q 'code/gendata.C(${events},"${gendata_output_file}")' | tee gendata.log
+            - root -b -q 'code/fitdata.C("${gendata_output_file}","${output_file}")' | tee fitdata.log
+    outputs:
+      files:
+        - results/data.root
+        - results/plot.png
+        
 In case you are using CWL or Yadage workflow specifications:
```

Comment:
Please simplify the rest of README text as was done in the world population example.

Please add yourself to the list of contributors